File: src/main/java/net/lucgameshd/easymysql/serialization/UUIDSerializer.java
package net.lucgameshd.easymysql.serialization;

import com.fasterxml.jackson.core.JsonGenerator;
import com.fasterxml.jackson.databind.JsonSerializer;
import com.fasterxml.jackson.databind.SerializerProvider;

import java.io.IOException;
import java.util.UUID;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class UUIDSerializer extends JsonSerializer<UUID> {

    @Override
    public void serialize( UUID uuid, JsonGenerator jsonGenerator, SerializerProvider serializerProvider ) throws IOException {
        jsonGenerator.writeString( uuid.toString() );
    }
}

File: src/main/kotlin/net/lucgameshd/easymysql/serialization/UUIDSerializer.kt
package net.lucgameshd.easymysql.serialization

import com.fasterxml.jackson.core.JsonGenerator
import com.fasterxml.jackson.databind.JsonSerializer
import com.fasterxml.jackson.databind.SerializerProvider
import java.util.UUID

class UUIDSerializer : JsonSerializer<UUID>() {
    override fun serialize(value: UUID, gen: JsonGenerator, serializers: SerializerProvider) {
        gen.writeString(value.toString())
    }
}
